examples/cra-ts-kitchen-sink/.storybook/preview.ts

```diff
@@ -2,5 +2,6 @@ export const parameters = {
   options: {
     brandTitle: 'CRA TypeScript Kitchen Sink',
     brandUrl: 'https://github.com/storybookjs/storybook/tree/master/examples/cra-ts-kitchen-sink',
+    brandTarget: '_self',
```

Comment:
We don't want to change our examples from default behavior, so this change should be removed please.

Reply:
Removed unnecessary changes